nav2_behavior_tree/plugins/condition/transform_available_condition.cpp

```diff
@@ -72,6 +65,22 @@ BT::NodeStatus TransformAvailableCondition::tick()
   return BT::NodeStatus::FAILURE;
 }
 
+void TransformAvailableCondition::initialize()
```

Comment:
Move function above tick

Reply:
^ do this for all nodes, just do I don't repeat this a ton of times

Reply:
done